include/opt-sched/Scheduler/machine_model.h

```diff
@@ -122,8 +127,8 @@ class MachineModel {
   // Returns the instruction type given the name of the instruction as well
   // as the name of the previous instruction (used for context-dependent
   // instructions).
-  InstType GetInstTypeByName(const std::string &typeName,
-                             const std::string &prevName = "") const;
+  InstType GetInstTypeByName(llvm::StringRef typeName,
+                             const string &prevName = "") const;
```

Comment:
```suggestion
                             const std::string &prevName = "") const;
```
Don't know why this was removed, this did not compile for me.